eis_toolkit/feature_importance/feature_importance.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+import pandas
```

Comment:
We have used `import pandas as pd` so far (and I noticed there seems to be duplicate import). Sorry for nitcpicky comment, but if its ok you should use the same import alias as elsewhere